src/io/cosmosoftware/kite/Utils.java

```diff
@@ -242,52 +235,56 @@ public static JsonArray checkStatus(String filePath) {
   }
 
   public static JsonObject countStatus(String filePath) {
-    String allureDirectory = null;
-    String osName = System.getProperty("os.name").toLowerCase();
-    if (osName.indexOf("win") >= 0) {
-      allureDirectory = filePath + "\\data\\test-cases\\";
-    } else if (osName.indexOf("nix") >= 0 || osName.indexOf("nux") >= 0 || osName.indexOf("aix") > 0) {
-      allureDirectory = filePath + "/data/test-cases/";
-    }
-    File allureFolder = new File(allureDirectory);
     JsonObjectBuilder jsonBuilder = Json.createObjectBuilder();
-    Map<String, Integer> countCases = new HashMap<String, Integer>();
+    String allureDirectory = filePath + (isWindowsBased() ? "\\data\\" : "/data/");
     try {
-      for (File subFile : allureFolder.listFiles()) {
-        JsonObject result = Utils.readJsonFile(subFile.getAbsolutePath());
-        countCases.merge(result.getString("status"), 1, Integer::sum);
+      File file=new File(allureDirectory + "suites.csv");
+      FileInputStream fis = new FileInputStream(file);
+      byte[] bytesArray = new byte[(int)file.length()];
+      fis.read(bytesArray);
+      String s = new String(bytesArray);
+      String [] passed = s.split("\"passed\"");
+      jsonBuilder.add("passed", passed.length - 1);
```

Comment:
needs  to check the length, otherwise display passed:0

Reply:
i know, added it